db/migrate/20151208150956_fix_host_storage_replication_on_upgrade.rb

```diff
@@ -9,36 +8,10 @@ class HostsStorage < ActiveRecord::Base
   end
 
   def up
-    if on_replication_target?
-      run_for_replication_target
-    else
-      run_for_replication_source
-    end
+    HostsStorage.delete_all if on_replication_target?
```

Comment:
So there is a way using `pglogical.alter_subscription_synchronize` to resync a specific subscription's data for a particular table, but that won't actually help us here because there is no way that pglogical will be enabled when this migration is being run.

The procedure for going from rubyrep to pglogical includes running `rake evm:dbsync:uninstall` on all the regional databases. We remove all of the rubyrep specific data  on the regions in the migrations in this PR (effectively a `rake evm:dbsync:local_uninstall`). Because of these facts, does it make sense to remove all regional data from the global in another migration in this PR instead of leaving it in a bad state or making the customer do it themselves (as we have removed the `evm:dbsync:uninstall` command in this PR)?

@Fryguy ?